docs/tutorial.rst

```diff
@@ -512,8 +512,9 @@ This provides both a ``size_history`` plot and a ``tubes`` visualization. E.g.
 .. code-block:: python
 
     import demesdraw
+
     graph = model.model.to_demes()
-    demesdraw.tubes(model.model.to_demes());
+    demesdraw.tubes(model.model.to_demes())
```

Comment:
do you actually want that change? in jupyter notebooks anyhow, the `;` makes it not print out the returned like <matplotlib.pyplot> object? (I'm not checking the compiled docs...)

Reply:
I changed that because `black` was otherwise complaining and suggested this change.

Reply:
These code blocks aren't actually being run during doc compilation though? So it shouldn't matter